contrib/pom.xml

```diff
@@ -477,7 +477,14 @@
     <dependency>
       <groupId>com.datastax.cassandra</groupId>
       <artifactId>cassandra-driver-core</artifactId>
-      <version>2.0.2</version>
+      <version>2.1.8</version>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
protocol version 3 was introduced in New in Java driver 2.1.2, that's why upgrading cassandra driver version.